### ISSUE TITLE
Add support for session that only has pycontrol data

### ISSUE THREAD

**Issue by teristam** [juliencarponcy/trialexp#42]
Currently a session must have both pycontrol and pyphotometry data

**Comment by teristam** [comment]
The current `00_create_session_folders.py` should already support this. And all pycontrol analysis should run without problem on data without photometry data already

**Comment by juliencarponcy** [comment]
Hey,

Did you re-open regarding synchronization on pycontrol instead of pyphotometry?

**Comment by teristam** [comment]
> Hey,
> 
> Did you re-open regarding synchronization on pycontrol instead of pyphotometry?

No, I thought this was implemented in the current pipeline, but apparently, it didn't. I have enabled copying of sessions that only have pycontrol data in my branch. I will submit an PR for that soon.
